CMakeLists.txt

```diff
@@ -15,11 +15,26 @@ else()
     set(CMAKE_CXX_FLAGS "-Werror -Wno-missing-declarations -O3 -fPIC")
 endif()
 
-#TODO the rest of deps should be moved outside of the project too (if possible)
+# TODO the rest of deps should be moved outside of the project too (if possible)
 MESSAGE("using deps from vcpkg or system")
 find_package(Boost 1.75.0 REQUIRED)
 find_package(OpenSSL REQUIRED)
-find_package(cryptopp CONFIG REQUIRED)
+find_package(CryptoPP REQUIRED)
+
+# TODO remove this logging
+message("CRYPTOPP_LIBRARIES: " ${CRYPTOPP_LIBRARIES})
```

Comment:
I guess it is remove ? :)

Reply:
@TomaszAugustyn will check if everything works correct tomorrow, then please remove.

Reply:
Removed.